pkg/geoipupdate/config_test.go

```diff
@@ -22,7 +22,7 @@ func TestNewConfig(t *testing.T) {
 	}{
 		{
 			Description: "Default config",
-			Input: `# Please see https://dev.maxmind.com/geoip/geoipupdate/ for instructions
+			Input: `# Please see https://dev.maxmind.com/geoip/updating-databases?lang=en for instructions
```

Comment:
Do we want to update the default config? `conf/GeoIP.conf.default`

Reply:
Good catch.